bitnami/grafana-mimir/values.yaml

```diff
@@ -0,0 +1,4185 @@
+        insecure: {{ not .Values.minio.tls.enabled }}
+      {{- else }}
+      backend: {{.Values.mimir.blockStorage.backend }}
+      {{ .Values.mimir.blockStorage.backend }}:
+        {{- include "common.tplvalues.render" (dict "value" .Values.mimir.blockStorage.config "context" $) | nindent 4 }}
+      {{- end }}
+      tsdb:
+        dir: {{ .Values.mimir.dataDir }}/tsdb
+    ingester:
+      compaction_interval: 30m
+      deletion_delay: 2h
+      max_closing_blocks_concurrency: 2
+      max_opening_blocks_concurrency: 4
+      symbols_flushers_concurrency: 4
+      data_dir: {{ .Values.mimir.dataDir }}/ingester
+      sharding_ring:
+        wait_stability_min_duration: 1m
+    compactor:
+      data_dir: {{ .Values.mimir.dataDir }}/compactor
+    frontend:
+      parallelize_shardable_queries: true
+      {{- if .Values.memcachedfrontend.enabled }}
+      results_cache:
+        backend: memcached
+        memcached:
+          timeout: 500ms
+          addresses: {{ include "grafana-mimir.memcached-frontend.host" . }}
+      cache_results: true
+      {{- end }}
+      {{- if .Values.queryScheduler.enabled }}
+      scheduler_address: {{ template "grafana-mimir.query-scheduler.fullname" . }}-headless.{{ .Release.Namespace }}.svc:{{ .Values.queryScheduler.service.ports.grpc }}
+      {{- end }}
+    frontend_worker:
+      grpc_client_config:
+        max_send_msg_size: 419430400 # 400MiB
+      {{- if .Values.queryScheduler.enabled }}
+      scheduler_address: {{ template "grafana-mimir.query-scheduler.fullname" . }}-headless.{{ .Release.Namespace }}.svc:{{ .Values.queryScheduler.service.ports.grpc }}
+      {{- else }}
+      frontend_address: {{ template "grafana-mimir.query-frontend.fullname" . }}-headless.{{ .Release.Namespace }}.svc:{{ .Values.queryFrontend.service.ports.grpc }}
+      {{- end }}
+    ingester:
+      ring:
+        final_sleep: 0s
+        num_tokens: 512
+        tokens_file_path: {{ .Values.mimir.dataDir }}/tokens
+        unregister_on_shutdown: false
+    ingester_client:
+      grpc_client_config:
+        max_recv_msg_size: 104857600
+        max_send_msg_size: 104857600
+    limits:
+      # Limit queries to 500 days. You can override this on a per-tenant basis.
+      max_total_query_length: 12000h
+      # Adjust max query parallelism to 16x sharding, without sharding we can run 15d queries fully in parallel.
+      # With sharding we can further shard each day another 16 times. 15 days * 16 shards = 240 subqueries.
+      max_query_parallelism: 240
+      # Avoid caching results newer than 10m because some samples can be delayed
+      # This presents caching incomplete results
+      max_cache_freshness: 10m
+    memberlist:
+      abort_if_cluster_join_fails: false
+      compression_enabled: false
+      join_members:
+      - dns+{{ include "grafana-mimir.gossip-ring.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.mimir.gossipRing.service.ports.http }}
```

Comment:
It is not clear to me if the gossipRing port can actually be changed.

Reply:
It can be changed now